t32/main.c

```diff
@@ -226,18 +239,35 @@ static uint32_t read_u32(const char *const name)
 int main(const int argc, const char *const argv[argc])
 {
   /* Trivial getopts */
-  if (argc != 3)
+  if (argc != 4)
   {
-    ERR("Usage: %s <script.cmm> <output.bin>", argv[0]);
+    ERR("Usage: %s <script.cmm> <output.bin> <logfile>", argv[0]);
     return 1;
   }
   const char *const script = argv[1];
   const char *const output = argv[2];
+  const char *const logfile = argv[3];
+  struct timeval stop, start;
+  const char *const envval = getenv("STUBBORN_MAX_MEASURES");
+  static uint8_t *BUFFER;
+  unsigned int buff_len = 1024;
+  /* We have a buffer of n elements (1024 by default), each element being a
+   * structure of size 24.  These are hard-coded parameters, not that great, but *
+   * it will do */
+  if(envval)
+  {
+    if(sscanf(envval, "%d", &buff_len) != 1)
+    { buff_len = 1024; }
+  }
+
+  BUFFER = calloc(buff_len, BUFF_EL_SIZE);
```

Comment:
Ideally, you should check against NULL and free this (don't bother, the `DIE()` macro makes this a nightmare).